test/wveOLAS.js

```diff
@@ -0,0 +1,509 @@
+/*global describe, context, beforeEach, it*/
+
+const { expect } = require("chai");
+const { ethers } = require("hardhat");
+
+describe("Wrapped Voting Escrow OLAS", function () {
```

Comment:
Tests are reduced in the next PR, since there is no need to repeat ve-only tests.